lib/mimdb_web/controllers/movie_controller.ex

```diff
@@ -4,9 +4,10 @@ defmodule MimdbWeb.MovieController do
   alias Mimdb.Movies
   alias Mimdb.Movies.Movie
 
-  def index(conn, _params) do
-    movies = Movies.list_movies()
-    render(conn, "index.html", movies: movies)
+  def index(conn, params) do
+    changeset = Movies.change_movie(%Movie{})
```

Comment:
I don't think this is used...did I miss something or can this be removed?

Reply:
Good catch @mratliff, thank you!